Apps/Src/ReadTritium.c

```diff
@@ -145,26 +156,26 @@ static inline void handler_Tritium_HallError(void) {
  * @param   motor_err Bitmap with motor error codes to check
  */
 static void assertTritiumError(tritium_error_code_t motor_err){   
-	static uint8_t hall_fault_cnt = 0; //trip counter, doesn't ever reset
+	static uint8_t motor_fault_cnt = 0; //trip counter, doesn't ever reset
 
 	Error_ReadTritium = (error_code_t)motor_err; // Store error code for inspection info
 	if(motor_err == T_NONE) return; // No error, return
 	
-	if(motor_err != T_HALL_SENSOR_ERR){
+	if(motor_err != T_HALL_SENSOR_ERR && motor_err != T_MOTOR_WATCHDOG_TRIP){
```

Comment:
Could it be the case that both happen? If that's the case, should it be `!((motor_err & T_HALL_SENSOR_ERR) || (motor_err & T_MOTOR_WATCHDOG_TRIP))`

Reply:
Good point, I'll fix that